app/controllers/application_controller.rb

```diff
@@ -34,13 +34,17 @@ def adequate_scopes?
     required_scopes.all? { |scope| current_scopes.include?(scope) }
   end
 
+  def request_scopes
+    required_scopes | current_scopes
+  end
```

Comment:
So when you sign into an invitation this is what this method looks like

```ruby
required_scopes
#=> ["user:email"]

# Because we aren't logged in yet.
current_scopes
#=> []

required_scopes | current_scopes
#=> ["user:email"]
```